Assets/Mirage/Runtime/ServerObjectManager.cs

```diff
@@ -204,34 +216,6 @@ public bool ReplaceCharacter(INetworkPlayer player, NetworkClient client, GameOb
             return InternalReplacePlayerForConnection(player, client, character, keepAuthority);
         }
 
-        void SpawnObserversForConnection(INetworkPlayer player)
```

Comment:
what replaces SpawnObserversForConnection? how do we spawn objects when the Player's character or scene changes?

Reply:
No direct replacement.  

When the scene changes, the objects will be spawned.  The interest manager will see that and show the object to all players.

when the player's character changes,  the interest manager should eventually refresh all objects.